doc/client.rst

```diff
@@ -2110,6 +2117,10 @@ Batch Apply Policies
             | Note that the TTL value will be employed ONLY on write/update calls.
             |
             | Default: 0
+        * **durable_delete** :class:`bool`
+            | If the transaction results in a record deletion, leave a tombstone for the record. This prevents deleted records from reappearing after node failures. Valid for Aerospike Server Enterprise Edition only.
+            |
+            | Default: :py:obj:`False` (do not tombstone deleted records).
```

Comment:
```suggestion
            | Default: :py:obj:`False` (do not tombstone delete records).
```

Reply:
nit

Reply:
The grammar looks correct

Reply:
got it, it means. do not place a tombstone after a record is deleted.